### PR TITLE
Added Survey API and Survey Results layout

### PR DESCRIPTION
**Pull request by Dez1999**
Added API for Survey (Controller, Service, Repo) and added Survey Results layout

### CODE REVIEW

File: src/main/resources/templates/surveyResults.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Survey Results</title>
+    <style>
```
Comment (keving3ng) [inline src/main/resources/templates/surveyResults.html:6]: Styles should be in an external css file

File: src/main/resources/templates/takerSurveyPage.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Individual Survey</title>
+    <style>
```
Comment (keving3ng) [inline src/main/resources/templates/takerSurveyPage.html:6]: same comment about stylesheets

**Review by keving3ng — COMMENTED** [review]
Styling should be moved to external css files but otherwise LGTM

**Review by keving3ng — APPROVED** [review]

**Review by OmarAgamy9 — APPROVED** [review]
Looks good to me

**Pull request merged**